helm-charts/flink-historyserver/templates/efs.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs-flink-history
```

```suggestion
  name: {{ .Release.Name }}-efs-flink-history
```

Since `StorageClass` is not namespaced, we need to differentiate these by name so multiple helm chart releases can be on the same cluster.

---

solved this